src/txpasslib/_threads.py

```diff
@@ -0,0 +1,24 @@
+from twisted.internet.defer import Deferred
+from twisted.python.failure import Failure
+
+
+def defer_to_worker(deliver, worker, work, *args, **kwargs):
```

Comment:
I would like to reuse the names from `passlib` where relevant, which means those will be `pep8_names`, and I think it is more consistent to then use that style everywhere.